TusStorageHandler/app.py

```diff
@@ -0,0 +1,115 @@
+"""Main application module for handling MinIO storage and TUS protocol with Flask."""
+
+import datetime
+import os
+import uuid
+
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from minio.error import S3Error
+from minio_client import minio_client
+from tus_server import TusServer
+from werkzeug.utils import secure_filename
+
+MINIO_BUCKET = os.getenv("MINIO_BUCKET_NAME")
+
+app = Flask(__name__)
+CORS(app, resources={r"/*": {"origins": "http://localhost:3000"}})
+
+tus_server = TusServer(minio_client, MINIO_BUCKET)
+
+
+@app.route("/", methods=["GET"])
```

Comment:
If you're writing an API why not use [FOCA](https://github.com/elixir-cloud-aai/foca/)??

Reply:
Yes, I plan to upgrade it using FOCA. First, I want to create a basic template to test it.